application-blog-ui/src/main/resources/Blog/Translations.fr.xml

```diff
@@ -121,56 +118,50 @@ blog.code.published=Cet article de blog n'a pas encore été publié.
 blog.code.readpost=Lire l’article complet
 blog.code.sheetexplanation=Cette sheet doit être utilisée afin d'afficher un article de blog.
 blog.code.title=Blog
-blog.code.warning=Attention \: 
-blog.events.blogpostpublished.by=par {0}. 
+blog.code.warning=Attention : 
+blog.events.blogpostpublished.by=par {0}.
 blog.events.blogpostpublished.description=Un billet a été publié
-blog.events.blogpostpublished.displayer=Un nouveau billet a été publié \: {0}
+blog.events.blogpostpublished.displayer=Un nouveau billet a été publié : {0}
 blog.livetable._actions=Actions
 blog.livetable.title=Titre
 blog.manage.blogtitle=Titre du Billet
-blog.manage.blogtype=Type de blog \:
-blog.manage.categoriesLocationPreview.hint=Emplacement dans la hiérarchie des pages où créer ces nouvelles catégories.
```

Comment:
is it normal that this key has been deleted?

Reply:
Actually it's been put below and the order changed. Why? Is that normal? (doesn't seem so)

Reply:
For example the keys were ordered before and it's no longer the case